homeassistant/components/srp_energy/const.py

```diff
@@ -0,0 +1,15 @@
+"""Constants for the SRP Energy integration."""
+from datetime import timedelta
+
+SRP_ENERGY_DOMAIN = "srp_energy"
```

Comment:
A bit overkill to rename it here as it only matters for the entity platforms. But ok :)

Reply:
Our standard is to name this constant `DOMAIN`.